lua/laravel/defaults.lua

```diff
@@ -22,6 +23,13 @@ local config = {
   lsp_server = "phpactor",
   route_info = true,
   default_runner = "buffer",
+  exec = {
+    npm = "vendor/bin/sail",
```

Comment:
this is not accurate since is not the executable is through which will be executed.

Reply:
Naming is bad - agreed, however it is kind of hard to give it a name, as we have runners, commands etc already kind of booked here.

Reply:
I think we could think of `docker-compose` and `sail` or `local` as `environments` because even in any container you will still run `npm` is not that expected to run. it will be up to the environment to define more to the command or not.
The environment could be define on the environment variable like .env, global configuration or autodetected(like now)

Reply:
I think we should then allow providing a custom environment or something, but it works for me in this way as well.